layouts/index.html

```diff
@@ -65,7 +66,11 @@ <h2 class="title--answer">Mostly...</h2>
       <li class="list__item--ok">
         Image viewer:
         <a href="https://wiki.gnome.org/Apps/EyeOfGnome">GNOME Image Viewer (eog)</a>,
-        <a href="https://github.com/eXeC64/imv">imv</a>
+        <a href="https://invent.kde.org/graphics/gwenview">Gwenview</a>,
```

Comment:
I am not a KDE expert, I don't use it but it looks like we are listing 3 KDE apps here, potentially doing the same thing. Can we list only 1 instead? If not, what is the difference between them?

Overall I think I want to remove all KDE and Gnome apps entirely. I would just create a new `Desktop environment` category and list the 2, it would imply that apps from those DE are WL native.

Reply:
Gwenview is desktop-oriented, Koko and Pix are touch/mobile oriented, but yes, all three of them are image viewers. If you want just one, then probably Gwenview is the best to keep there since it works on touch even if the interface is made for desktop use.

It might be a good idea to clarify in the readme that you want a non-comprehensive list of apps composed of mostly third-party/DE or WM agnostic apps (so no default GNOME, Plasma or Sway tools). I think in the next few years XFCE, Mate and Pantheon will also join the list of DEs, as all three have plans for that (and Budgie is just waiting for all needed protocols to be done).